src/app/public/testing/phonefield/phone-field-fixture.ts

```diff
@@ -0,0 +1,182 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldFixture
+} from '@skyux/lookup/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX phone field component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyPhoneFieldFixture {
+  private _debugEl: DebugElement;
+
+  // this property is lazy loaded and should be accessed via the private countryFixture property
+  private _countryFixture: SkyCountryFieldFixture;
+
+  /**
+   * The value of the input field for the phone field.
+   */
+  public get inputText(): string {
+    return this.phoneFieldInput.value;
+  }
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this._debugEl = SkyAppTestUtility
+      .getDebugElementByTestId(fixture, skyTestId, 'sky-phone-field');
+
+    /* Attempt #1:
+      This doesn't work because the phone-field only has commented out child elements
+      */
+    // // tag the country field with a sky test id
+    // const countrySkyTestId = `${skyTestId}-country`;
+    // this.setSkyTestId(this.countryElement, countrySkyTestId);
+
+    // // grab the country field
+    // this._countryFixture = new SkyCountryFieldFixture(fixture, countrySkyTestId);
+
+    /* Attempt #2:
+      This doesn't work because the country-field takes too long to initialize.
+      It introduces a race condition when trying to access the country element will throw a null reference.
+      */
+    // fixture.detectChanges();
+    // fixture.whenStable().then(() => {
+    //   const debugEl = this._debugEl;
+
+    //   // tag the country field with a sky test id
+    //   const countrySkyTestId = `${skyTestId}-country`;
+    //   this.setSkyTestId(this.countryElement, countrySkyTestId);
+
+    //   // grab the country field
+    //   this._countryFixture = new SkyCountryFieldFixture(fixture, countrySkyTestId);
+    // });
+
+    /* Attempt #3:
```

Comment:
Note the 3 attempts to initialize the country field inside the constructor. My final attempt worked but it still seems to have a race condition, so I wanted to be safe and avoid it entirely.